recipes/nansat/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set name = "nansat" %}
+{% set version = "0.6.17" %}
+{% set sha256 = "ecaa305250d3a34ad7d4922034bf33731ec0da127afa533cbb74728a69bab2dd" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+ fn: v{{ version }}.tar.gz
+ url: https://github.com/nansencenter/nansat/archive/v{{ version }}.tar.gz
+ sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [osx or not py27]
+
+requirements:
+  build:
```

Comment:
Are most of the build requirements actually needed? If they're only in `install_requires`, `--single-version-externally-managed` means they don't need to be present at build time.

Reply:
Unfortunately most of them are not only in install_requires in this version. Someone had the brilliant idea to e.g. write code like this in setup.py:

```
try:
    import scipy
except ImportError:
    raise ImportError(import_error_msg % 'scipy')
```

This is fixed in the newer version, but that one is not production ready yet. I will clean this up when the new version if production ready and we can publish it on conda-forge.

Reply:
You could always patch the `setup.py`

Reply:
It's not a huge deal; it just slows down builds a bit, and raises a little bit of risk that the compiled code links against something in `scipy` (which would then have runtime version implications). I don't think that's happening here, so not worth patching `setup.py` to work around it IMO.